tests/BulkTest.php

```diff
@@ -135,4 +135,12 @@ public function testItShouldRaiseAnExceptionIfTheErrorListIsNotEmpty()
         $this->bulk->addItem('idFoo', 'foo');
         $this->bulk->flush();
     }
+
+    public function testItShouldEmptyTheItemListOnClear()
+    {
+        $this->bulk->addItem('idfoo','foo');
+        $this->assertEquals($this->bulk->getItemCount(), 1);
+        $this->bulk->clear();
+        $this->assertEquals($this->bulk->getItemCount(), 0);
```

Comment:
You can assert that the item list is empty as well with `assertAttributeEquals` (I think that's the method name -- something close to that at least).